src/Symfony/Component/Notifier/Tests/Transport/NullTransportTest.php

```diff
@@ -31,7 +31,7 @@ public function testSend()
             $eventDispatcherMock = $this->createMock(EventDispatcherInterface::class)
         );
 
-        $eventDispatcherMock->expects($this->once())->method('dispatch');
+        $eventDispatcherMock->expects($this->exactly(2))->method('dispatch');
         $nullTransport->send(new DummyMessage());
     }
 }
```

Comment:
Could you please add dedicated tests for the events? Thanks

Reply:
yes I'll add tests ty